mlflow_openshift/deployment_client.py

```diff
@@ -1,249 +1,249 @@
-import json
```

Comment:
please, keep all files back to LF (here nice article about LF and CRLF https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/ )

Reply:
ah ok, I see your point. I reverted the line endings as desired. ok?